en/identity-server/7.0.0/docs/guides/users/manage-users.md

```diff
@@ -67,6 +67,9 @@ In addition to adding a single user, you can onboard multiple users at once, eit
 
 #### Add multiple users using a CSV file
 
+!!! Info
+    The maximum number of users that can be added via a CSV file is limited to 100.
```

Comment:
this limit is for "per attempt" right?

Reply:
Yes, a CSV file can be attached per request

Reply:
Shall we update the content to make it clear?